allennlp/interpret/saliency_interpreters/smooth_gradient.py

```diff
@@ -41,13 +43,14 @@ def _register_forward_hook(self, stdev: int):
         Register a forward hook on the embedding layer which adds random noise to every embedding.
         Used for one term in the SmoothGrad sum.
         """
-        def forward_hook(module, input, output):
+        def forward_hook(_, _, output):
```

Comment:
On these (there's another one in a different part of the code), I'd actually recommend keeping the names and disabling pylint, so the reader isn't left wondering what these unused arguments are.